README.md

```diff
@@ -45,6 +47,106 @@ load_dotenv()
 # assume that the MONGODB_NAME is not loaded
 # so we need to set it manuall
 session = MongoDBSession(db_name="my-db-name")
+
+```
+
+You can use the session to execute simple queries as shown below
+
+```commandline
+import asyncio
+import bson
+from navalmartin_mir_db_utils.dbs.mongodb_session import MongoDBSession
+from navalmartin_mir_db_utils.crud.mongodb_crud_utils import ReadEntityBaseCRUDAPI
+
+IMAGES_COLLECTION_TO_READ="YOUR_COLLECTION_NAME"
+
+
+async def query_db(mongodb_session: MongoDBSession, criteria: dict,
+                   projection: dict,
+                   collection_name: str):
+    query_result = ReadEntityBaseCRUDAPI.do_find(criteria=criteria, projection=projection,
+                                                 db_session=mongodb_session,
+                                                 collection_name=collection_name)
+    images = [img async for img in query_result]
+    return images
+    
+    
+def main():
+    MONGODB_URL = "YOUR_MONGODB_URL"
```

Comment:
don't you need to access the environmental variables here, eg `get.env("...")`?

Reply:
This is left to be specified by the client code how to do this